src/overlays/actors/ovl_En_Zo/z_en_zo.c

```diff
@@ -15,17 +15,289 @@ void EnZo_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnZo_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnZo_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06000598;
-extern UNK_TYPE D_06000D48;
-extern UNK_TYPE D_060022F0;
-extern UNK_TYPE D_06002350;
-extern UNK_TYPE D_060024A0;
-extern UNK_TYPE D_06002510;
-extern UNK_TYPE D_0600BFA8;
-extern UNK_TYPE D_0600BFC0;
-extern UNK_TYPE D_0600C028;
+// Actions
+void EnZo_Standing(EnZo* this, GlobalContext* globalCtx);
+void EnZo_Submerged(EnZo* this, GlobalContext* globalCtx);
+void EnZo_Surface(EnZo* this, GlobalContext* globalCtx);
+void EnZo_TreadWater(EnZo* this, GlobalContext* globalCtx);
+void EnZo_Dive(EnZo* this, GlobalContext* globalCtx);
+
+extern Gfx D_060022F0[];
+extern Gfx D_06002350[];
+extern Gfx D_060024A0[];
+extern Gfx D_06002510[];
+extern Gfx D_0600BFC0[];
+extern Gfx D_0600C028[];
+extern FlexSkeletonHeader D_0600BFA8;
+
+extern AnimationHeader D_06002FE8;
+extern AnimationHeader D_06002F10;
+extern AnimationHeader D_0600219C;
+extern AnimationHeader D_06000598;
+extern AnimationHeader D_06000D48;
+
+// Surface Ripples
+void EnZo_Ripple(EnZo* this, Vec3f* pos, f32 scale, f32 targetScale, u8 alpha) {
+    EnZoEffect* effect;
+    Vec3f vec = { 0 };
+    s16 i;
+
+    effect = this->effects;
+    for (i = 0; i < ARRAY_COUNT(this->effects); i++) {
+        if (effect->type == 0) {
+            effect->type = 1;
+            effect->pos = *pos;
+            effect->scale = scale;
+            effect->targetScale = targetScale;
+            effect->color.a = alpha;
+            return;
```

Comment:
break instead of return makes more sense here I think
```suggestion
            break;
```